release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -178,17 +178,42 @@ Limit Calculator] to estimate cluster limits for your environment.
 [id="ocp-4-3-storage"]
 === Storage
 
-[id="ocp-4-3-iscsi"]
-==== iSCSI
+[id="ocp-4-3-ocs-4-2"]
+==== OpenShift Container Storage 4.2
 
-Persistent volumes and raw block volumes using iSCSI, previously in Technology
-Preview, are now fully supported in {product-title} 4.3.
+Deploy, manage, monitor, and migrate a Red Hat OpenShift Container Storage 4.2 cluster. See the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.2/html/4.2_release_notes/[Red Hat OpenShift Container Storage 4.2 Release Notes].
```

Comment:
This first sentence isn't really a sentence. Should it be something like "You can now use OpenShift Container Storage 4.2 to deploy, manage, monitor, and migrate ... " actually I don't know what? OCP clusters?

I would also make the "See the release notes..." a full sentence too by adding "for more information" or something at the end.

Reply:
It's actually to deploy, manage an OCS cluster, so I added, "You can now..." at the beginning of the sentence.